lib/server.js

```diff
@@ -1,10 +1,33 @@
 const express = require('express')
 
-module.exports = function (webhook) {
+module.exports = function (webhook, logger) {
   const app = express()
 
+  app.use((req, res, next) => {
+    log(req, logger)
+    next()
+  })
   app.use(webhook)
   app.get('/ping', (req, res) => res.end('PONG'))
 
   return app
 }
+
+function log (req, logger) {
+  if (process.env.LOG_LEVEL === 'trace') {
+    logger.trace({
+      method: req.method,
+      path: req.path,
+      ip: req.ip,
+      hostname:
+      req.hostname,
```

Comment:
This should be on one line, would you mind making that fix please?

Reply:
Sure. I will take care of this later today